src/inputs/Address.tsx

```diff
@@ -39,7 +39,7 @@ class Address extends Component<IAddressProps> {
   }
 
   private get fieldSet () {
-    const { fieldConfig: { colProps, required, stateProps } } = this.injected
+    const { fieldConfig: { colProps, required, stateProps, disabled } } = this.injected
```

Comment:
I thought that `Address` was outside of this use case?

Reply:
It was easy to implement it for all types.